Ulm-IQO/qudi
# SI units for confocal microscope module

Please test thoroughly, especially saving and loading things.

Test instructions:
Before testing this branch, make sure to move your current app_status folder out of the way, otherwise you *will* get errors.
Also make sure to convert your scanner x_range, y_range and z_range to SI base units, in this case meters!
